src/lib/core/components/EDAWorkspaceContainer.tsx

```diff
@@ -15,7 +15,7 @@ import { workspaceTheme } from './workspaceTheme';
 const theme = createMuiTheme(workspaceTheme);
 export interface Props {
   studyId: string;
-  analysisId: string;
+  analysisId?: string;
```

Comment:
I don't see `analysisId` used in `EDAWorkspaceContainer` nor do I see `props` passed on to another component, so can this be removed completely?

Reply:
Yes! I will remove